recipes/wasmtime-cpp/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration, ConanException
+import os
+import shutil
+
+required_conan_version = ">=1.33.0"
+
+class WasmtimeCppConan(ConanFile):
+    name = 'wasmtime-cpp'
+    homepage = 'https://github.com/bytecodealliance/wasmtime-cpp'
+    license = 'Apache-2.0'
+    url = 'https://github.com/conan-io/conan-center-index'
+    description = "Standalone JIT-style runtime for WebAssembly, using Cranelift"
+    topics = ("webassembly", "wasm", "wasi", "c++")
+    settings = "compiler"
+    exports_sources = "include/*"
```

Comment:
`exports_soruces` is for copying sources from the recipe folder to the source folder.
Because I don't see any header files added here, this is not needed/used.
```suggestion

```

Reply:
Done